test/cctest/node_test_fixture.h

```diff
@@ -53,72 +53,61 @@ struct Argv {
   int nr_args_;
 };
 
-extern uv_loop_t current_loop;
 
 class NodeTestFixture : public ::testing::Test {
- public:
-  static uv_loop_t* CurrentLoop() { return &current_loop; }
-
-  node::MultiIsolatePlatform* Platform() const { return platform_; }
-
  protected:
+  static std::unique_ptr<v8::ArrayBuffer::Allocator> allocator;
+  static std::unique_ptr<v8::TracingController> tracing_controller;
+  static std::unique_ptr<node::NodePlatform> platform;
+  static v8::Isolate::CreateParams params;
+  static uv_loop_t current_loop;
   v8::Isolate* isolate_;
 
-  ~NodeTestFixture() {
-    TearDown();
-  }
-
-  virtual void SetUp() {
+  static void SetUpTestCase() {
+    platform.reset(new node::NodePlatform(8, nullptr));
```

Comment:
Maybe lower this to one to make the tests more deterministic.

As well, different cctests can (at least in theory) run concurrently; you don't want to have an explosion of threads.

Reply:
Sounds good, I'll change it to 4.